docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -4,13 +4,17 @@
 
 beta::[]
 
-Phases allowed: cold.
+Phases allowed: hot, cold.
 
 Takes a snapshot of the managed index in the configured repository
 and mounts it as a searchable snapshot.
 If the managed index is part of a <<data-streams, data stream>>,
 the mounted index replaces the original index in the data stream.
 
+IMPORTANT: If the `searchable_snapshot` action is used in the `hot` phase the
+subsequent phases cannot define any of the `shrink`, `forcemerge`, `freeze` or
+`searchable_snapshot` (also available in the cold phase) actions.
```

Comment:
Can you add a blurb similar to what we have for forcemerge related to requiring the "rollover" action:

"To use the XYZ action in the hot phase, the rollover action must be present. If no rollover action is configured, ILM will reject the policy."